tools/js-sdk-release-tools/src/hlc/utils/automaticGenerateChangeLogAndBumpVersion.ts

```diff
@@ -18,15 +18,35 @@ import {isGeneratedCodeStable} from "./isGeneratedCodeStable";
 import {execSync} from "child_process";
 import { getversionDate } from "../../utils/version";
 
-const fs = require('fs');
-const path = require('path');
+import fs from 'fs';
+import path from 'path';
+import shell from 'shelljs';
+import { exit } from "process";
+
+enum CodeGenLevel {
+    Classic = 'Classic',
+    Modular = 'Modular',
+};
+
+function getCodeGenLevel(parametersPath: string) : CodeGenLevel {
```

Comment:
a little wired if we have parametersPath as param here. because for different sdk the parameter path is fixed.